src/webview/common/componentNameInput.tsx

```diff
@@ -9,21 +9,23 @@ export type ComponentNameInputProps = {
     isComponentNameFieldValid: boolean,
     componentNameErrorMessage: string,
     componentName: string,
-    setComponentName: React.Dispatch<React.SetStateAction<string>>
+    setComponentName: React.Dispatch<React.SetStateAction<string>>,
+    label?: string,
+    action?: string
 };
 
 export function ComponentNameInput(props: ComponentNameInputProps) {
     return (
         <TextField fullWidth
-            id='componentName'
+            id={`componentName-${Math.random()}`}
```

Comment:
This id will change every time the form is rerendered, since Math.random() will be called each time. Is there any reason to use a random number here? If it's being used to differentiate component name input fields, could one of the props be used instead of random number?